angular1/app/components/version/interpolate-filter.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
I do not think we need the .js files in the sample, it would be confusing i would say.. and they should be generated anyways. and the other samples do not have them